configs/eln_extras_meta.yaml

```diff
@@ -94,15 +93,19 @@ data:
 
   arch_packages:
     aarch64:
+      - awscli2
       - sedutil
       - sysbench
     i686:
+      - awscli2
```

Comment:
why is i686 here at all?  (otherwise looks good though)

Reply:
Uh, I actually don't know, I think I copied the structure from another config. We definitely don't need i686 for what we're using this config at Meta. Should I drop it?

Reply:
Yes, drop the entire i686 field here (probably makes more sense as a separate commit as part of this PR).